frontend/src/views/Pages/Register.vue

```diff
@@ -86,13 +86,14 @@
 
                   <b-row>
                     <b-col cols="12">
-                      {{ $t('language') }}
-                      <b-form-select
-                        id="selectedLanguage"
-                        v-model="selected"
-                        :options="options"
-                        class="mb-3"
+                      {{ $t('language') }} {{ language }}
+                      <!--   <b-form-select
+                        v-model="language"
+                        class="selectedLanguage mb-3"
```

Comment:
Can you remove the comment if this was the old implementation? Seems logical to me.